.gitattributes

```diff
@@ -1 +1,2 @@
 * text eol=lf
+bin/just-1.14.0-x86_64-unknown-linux-musl binary
```

Comment:
ask git not to corrupt the file

Reply:
huh, TIL. Should to that for docker-compose too?